indexd/index/drivers/alchemy.py

```diff
@@ -905,9 +902,18 @@ def migrate_7(session, **kwargs):
             session.delete(metadata)
 
 
+def migrate_8(session, **kwargs):
+    """
+    create index on IndexRecord.baseid
+    """
+    session.execute(
+        "CREATE INDEX ix_{tb}_baseid ON {tb} ( baseid )"
+        .format(tb=IndexRecord.__tablename__))
+
+
```

Comment:
Should check if `ix_index_record_baseid` index exist before create a new one

Reply:
None of the existing migration check that because it's relying on the migration version table